server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -762,6 +771,20 @@ public String serviceName(ServiceRequestContext ctx) {
         }
     }
 
+    @Nullable
+    private static Function<? super HttpService, CorsService> configCorsDecorator(List<String> allowedOrigins) {
+        if (allowedOrigins == null || allowedOrigins.isEmpty()) {
+            return null;
+        }
+
+        return CorsService.builder(allowedOrigins)
+                       .allowRequestMethods(HttpMethod.knownMethods())
+                       .allowAllRequestHeaders(true)
+                       .allowCredentials()
+                       .maxAge(1800)
```

Comment:
Where did `1800` come from? We may expose the value as a configuration and could note the default value in the documentation.

I prefer to follow the default value of Chrome which is the most popular browser although I'm using Firefox. 🦊
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age#directives

Reply:
The JSON configuration may need changing like:
```json
"cors": {
   // It should be nice if we both support string and array of strings.
   "allowedOrigins": "*",
   "allowedOrigins": ["foo.com", "bar.com"],
   "maxAge": 1800  // optional. If not set, 7200 is used by default?
}
```

Reply:
i follow spring default configuration at [https://github.com/spring-projects/spring-framework/blob/3854861a8a71da97a8275c76ea81ad04025da9ae/spring-web/src/main/java/org/springframework/web/cors/CorsConfiguration.java#L499](url). Will make this configurable as you suggest